src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result = result - 50;
+  } else if (days >= 3) {
+    result = result - 20;
```

Comment:
Similarly, use a constant for the number of days that qualifies for the short-term discount, and another constant for the amount of the discount. Also, remove the `else` keyword as per checklist items 2 and 5, since the `if` block contains a `return` statement.